ui/app/src/context/Config.tsx

```diff
@@ -102,6 +103,17 @@ export function useIsAuthEnabled(): boolean {
   return config.security.enable_auth;
 }
 
+export function useIsNativeAuth(): boolean {
+  const { config } = useConfigContext();
+  return !!config.security.authorization.provider.native?.enable;
```

Comment:
In case this is not defined in the config
```suggestion
  return !!config.security.authorization.provider?.native?.enable;
```